Makefile

```diff
@@ -14,7 +14,7 @@ lint:
 depsdev:
 	go install github.com/Songmu/ghch/cmd/ghch@v0.10.2
 	go install github.com/Songmu/gocredits/cmd/gocredits@v0.2.0
-	go install github.com/securego/gosec/v2/cmd/gosec@v2.8.1
+	go install github.com/securego/gosec/v2/cmd/gosec@v2.11.0
```

Comment:
CI failed, so I fixed it with the following
https://github.com/securego/gosec/issues/797